### PR TITLE
Invites: Link site title to site in accepted notice

### LINKED ISSUE #2337: People: Link site title to site in accept invite notice
In [this comment](https://github.com/Automattic/wp-calypso/issues/2281#issuecomment-171015250), @rachelmcr suggested that we link the site title to the site within the invite accepted notice.

### PR DESCRIPTION
**Pull request by ebinnion**
Closes #2337

Previously, the site title was not linked, which may have made it difficult for users to see which site they were added to. This PR links the site title that in the invite accepted notice.

screen shot 5

cc @lezama

### CODE REVIEW

File: client/my-sites/invites/invite-accept/index.jsx [hunk 1]
```diff
@@ -249,6 +242,7 @@ let InviteAccept = React.createClass( {
 	},
 
 	render() {
+		console.log( this.state );
```
Comment (lezama) [inline client/my-sites/invites/invite-accept/index.jsx]: oops :)

Comment (ebinnion) [inline client/my-sites/invites/invite-accept/index.jsx]: :scream:

File: client/my-sites/invites/invite-accept/index.jsx [hunk 2]
```diff
@@ -88,15 +89,35 @@ let InviteAccept = React.createClass( {
...
 	},
 
+	clickedNoticeSiteLink() {
+		analytics.tracks.recordEvent( 'calypso_invite_accept_notice_site_link_click' );
+	},
+
 	acceptedNotice( invite ) {
 		invite = invite || this.state.invite;
 		let displayOnNextPage = true;
+		let takeATour = (
```
Comment (lezama) [inline client/my-sites/invites/invite-accept/index.jsx]: the three variables declared here could be `const` including `displayOnNextPage` that I introduced :blush:

**Comment by lezama** [comment]
LGTM!

**Pull request merged**